contract_payment_auto/README.rst

```diff
@@ -0,0 +1,96 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Can you separate the README sections into their own files in a README directory [as described here](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#tasks-to-do-in-the-migration)